shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -379,14 +392,67 @@ - (void)setFlutterViewDidRenderCallback:(void (^)(void))callback {
 
 - (void)surfaceUpdated:(BOOL)appeared {
   // NotifyCreated/NotifyDestroyed are synchronous and require hops between the UI and GPU thread.
+  [self setEnableForRunnersBatch:YES];
   if (appeared) {
     [self installSplashScreenViewCallback];
     [_engine.get() platformViewsController] -> SetFlutterView(_flutterView.get());
     [_engine.get() platformView] -> NotifyCreated();
   } else {
     [_engine.get() platformView] -> NotifyDestroyed();
     [_engine.get() platformViewsController] -> SetFlutterView(nullptr);
+    [self disableGPUOperation];
+  }
+}
+
+
+- (flutter::TaskRunners)getTaskRunners {
+  return ([_engine.get() shell].GetTaskRunners());
+}
+
+- (void)disableGPUOperation {
+  [[_engine.get() lifecycleChannel] sendMessage:@"AppLifecycleState.paused"];
+  //暂时通过延时来等待GL操作结束(否则进入后台后的GL操作会闪退)
```

Comment:
Can you guys change these comments to english? :D

Reply:
sure，we will change it later